control_server/tests/middleware/generic_message_test.py

```diff
@@ -0,0 +1,77 @@
+from typing import List, Tuple, Dict, Any
+
+import pytest
+
+from control_server.src.middleware.headers.message_header import MessageHeader
+from control_server.src.middleware.messages.generic_message import \
+    GenericMessage
+from control_server.tests.utils.header_utils import HeaderUtils
+
+datasets: List[Tuple[bytes, Dict[str, Any]]] = [
+    (
+        b'\x00\x00\x00\x00\x00\x00\x00\x00\x00AAA',
```

Comment:
In this case, it does not matter, because generic_message_test.py only tests that the message is serialized and deserialized correctly, and provides another header in the utils method, that sets URL, body and header length to one. This should however be improved in a future PR to clarify.

Reply:
Since the UDP communication part was not intended to be included in this PR, and since the tests still pass and the server as a whole works, this will be fixed in a future PR instead which contains a more complete version of the UDP server layer of the control server.